linux_os/guide/services/base/service_abrtd_disabled/rule.yml

```diff
@@ -1,6 +1,6 @@
 documentation_complete: true
 
-prodtype: ol7,ol8,ol9,rhel7,rhel8
+prodtype: uosv20,ol7,ol8,ol9,rhel7,rhel8
```

Comment:
This comment applies all of the changes in the `linux_os` directory.

Please run `./utils/fix_rules.py sort_prodtypes` to fix these rules. See [the docs](https://complianceascode.readthedocs.io/en/latest/manual/developer/05_tools_and_utilities.html#utils-fix-rules-py-automatically-fix-up-rules) for how to use this script.